bigbluebutton-html5/imports/ui/components/chat/styles.scss

```diff
@@ -12,6 +12,20 @@
   text-decoration: none;
 }
 
+.closeBtn {
+  position: absolute;
+  right: 10px;
+  top: 10px;
+}
+
+// Modifies the close button style
+Button.closeBtn span:first-child {
```

Comment:
I took that from another css file that was already there and using the same button for the close.